### PR TITLE
Actually drop duplicated messages

### PR DESCRIPTION
**Pull request by Pchelolo**
Actually begin dropping duplicate messages

cc @wikimedia/services

### CODE REVIEW

File: lib/rule_executor.js [hunk 1]
```diff
@@ -47,16 +46,8 @@ class RuleExecutor extends BaseExecutor {
...
-                this.log('warn/dedupe', {
-                    message: 'Event was deduplicated',
-                    event_str: utils.stringify(expander.message)
-                });
-            }
-        })
-        .catch({ status: 404 }, () => {})
-        .thenReturn(expander);
+        .get('body')
+        .catch({ status: 404 }, () => false);
```
Comment (gwicke) [inline lib/rule_executor.js]: Hmmm.. just `false` is not really a valid HTTP response. Perhaps make this `{ body: false }` instead?

Comment (Pchelolo) [reply lib/rule_executor.js]: But it's just a function call, we don't call it via http interface

Comment (gwicke) [reply lib/rule_executor.js]: Ah, right.. this is the caller, not the callee. Nevermind.

**Comment by gwicke** [comment]
LGTM, lets just wait for Mr. Travis.

**Comment by Pchelolo** [comment]
Ok travis done, merging,

**Pull request merged**